From 0ff36c191802898c7e7d7414b1feb461bf23d71d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc=20Poulhi=C3=A8s?= <dkm@kataplop.net>
Date: Mon, 12 May 2025 22:42:38 +0200
Subject: [PATCH] ci: adjust expected warnings after upstream sync
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

ChangeLog:

	* .github/alpine_32bit_log_warnings: Adjust with latest warnings.
	* .github/glibcxx_ubuntu64b_log_expected_warnings: Likewise.
	* .github/log_expected_warnings: Likewise.

Signed-off-by: Marc Poulhiès <dkm@kataplop.net>
---
 .github/alpine_32bit_log_warnings             |  91 +++++++-
 .../glibcxx_ubuntu64b_log_expected_warnings   | 177 ++++++++++-----
 .github/log_expected_warnings                 | 212 ++++++++++++------
 3 files changed, 351 insertions(+), 129 deletions(-)

diff --git a/.github/alpine_32bit_log_warnings b/.github/alpine_32bit_log_warnings
index 9c99c7c1402..7de91602a01 100644
--- a/.github/alpine_32bit_log_warnings
+++ b/.github/alpine_32bit_log_warnings
@@ -62,16 +62,89 @@
 ../../../libgcc/soft-fp/op-common.h:1563:25: warning: comparison of integer expressions of different signedness: ‘int’ and ‘USItype’ {aka ‘unsigned int’} [-Wsign-compare]
 ../../../libgcc/soft-fp/op-common.h:1563:25: warning: comparison of integer expressions of different signedness: ‘int’ and ‘USItype’ {aka ‘unsigned int’} [-Wsign-compare]
 ../../../libgcc/soft-fp/op-common.h:1563:25: warning: comparison of integer expressions of different signedness: ‘int’ and ‘USItype’ {aka ‘unsigned int’} [-Wsign-compare]
-../../gcc/../libgcc/libgcov-util.c:214:59: warning: 'void* calloc(size_t, size_t)' sizes specified with 'sizeof' in the earlier argument and not in the later argument [-Wcalloc-transposed-args]
-../../gcc/../libgcc/libgcov-util.c:529:43: warning: 'void* calloc(size_t, size_t)' sizes specified with 'sizeof' in the earlier argument and not in the later argument [-Wcalloc-transposed-args]
-../../gcc/analyzer/store.h:310:5: warning: 'size_in_bytes.generic_wide_int<fixed_wide_int_storage<128> >::fixed_wide_int_storage<128>.fixed_wide_int_storage<128>::val[1]' may be used uninitialized [-Wmaybe-uninitialized]
-../../gcc/expmed.cc:1845:45: warning: '*(unsigned int*)((char*)&imode + offsetof(scalar_int_mode, scalar_int_mode::m_mode))' may be used uninitialized [-Wmaybe-uninitialized]
-../../gcc/text-art/style.cc:150:25: warning: spurious leading punctuation sequence ';' in format [-Wformat-diag]
-../../gcc/text-art/style.cc:160:25: warning: spurious leading punctuation sequence ';' in format [-Wformat-diag]
-../../gcc/text-art/table.cc:981:62: warning: unquoted keyword 'char' in format [-Wformat-diag]
-../../gcc/text-art/table.cc:981:69: warning: spurious trailing punctuation sequence '])' in format [-Wformat-diag]
+../../gcc/analyzer/kf-analyzer.cc:202:40: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/analyzer/kf-analyzer.cc:202:55: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/analyzer/store.h:312:5: warning: 'size_in_bytes.generic_wide_int<fixed_wide_int_storage<128> >::fixed_wide_int_storage<128>.fixed_wide_int_storage<128>::val[1]' may be used uninitialized [-Wmaybe-uninitialized]
+../../gcc/attribs.cc:2330:26: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/attribs.cc:2331:44: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/attribs.cc:2332:26: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/attribs.cc:2333:44: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/attribs.cc:2352:22: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/attribs.cc:2353:40: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/attribs.cc:2354:22: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/attribs.cc:2355:40: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c-family/c-common.cc:2978:13: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c-family/c-common.cc:2978:51: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c-family/c-common.cc:2978:58: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c-family/c-format.cc:4910:24: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c-family/c-format.cc:4910:29: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c-family/c-format.cc:4910:57: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c-family/c-format.cc:4911:39: warning: format '%d' expects argument of type 'int', but argument 5 has type 'element_format_substring*' [-Wformat=]
+../../gcc/c-family/c-format.cc:4911:51: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c-family/c-format.cc:4917:24: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c-family/c-format.cc:4917:29: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c-family/c-format.cc:4917:51: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:7628:11: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:7628:22: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:7628:49: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:7727:21: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:7727:23: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8577:36: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:8577:52: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8578:39: warning: format '%D' expects argument of type 'tree_node*', but argument 4 has type 'pp_markup::element_expected_type*' [-Wformat=]
+../../gcc/c/c-typeck.cc:8578:65: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8583:36: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:8583:52: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8584:51: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8592:57: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8593:66: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8598:57: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8599:52: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8606:36: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:8606:71: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8607:72: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8612:36: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:8612:48: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8613:63: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8635:31: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:8635:33: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/diagnostic-path.cc:1190:29: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/diagnostic-path.cc:1190:34: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/diagnostic-path.cc:1190:46: warning: format '%s' expects argument of type 'char*', but argument 4 has type 'element_event_desc*' [-Wformat=]
+../../gcc/diagnostic-path.cc:1190:56: warning: format '%i' expects argument of type 'int', but argument 5 has type 'const char*' [-Wformat=]
+../../gcc/diagnostic-path.cc:1196:27: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/diagnostic-path.cc:1196:32: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/diagnostic-path.cc:1196:42: warning: format '%i' expects argument of type 'int', but argument 4 has type 'element_event_desc*' [-Wformat=]
+../../gcc/diagnostic-path.cc:1202:23: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/diagnostic-path.cc:1202:28: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/expmed.cc:1864:45: warning: '*(unsigned int*)((char*)&imode + offsetof(scalar_int_mode, scalar_int_mode::m_mode))' may be used uninitialized [-Wmaybe-uninitialized]
+../../gcc/gimple-ssa-warn-access.cc:2972:20: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gimple-ssa-warn-access.cc:2972:39: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/gimple-ssa-warn-access.cc:3023:22: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gimple-ssa-warn-access.cc:3023:41: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/opts-diagnostic.cc:173:10: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/opts-diagnostic.cc:173:10: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/opts-diagnostic.cc:174:61: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/opts-diagnostic.cc:174:61: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/opts-diagnostic.cc:254:6: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/opts-diagnostic.cc:255:53: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/opts-diagnostic.cc:357:26: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/opts-diagnostic.cc:358:69: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/pretty-print.cc:3559:19: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/pretty-print.cc:3559:28: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/pretty-print.cc:3559:38: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/pretty-print.cc:3698:19: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/pretty-print.cc:3698:28: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/pretty-print.cc:3698:38: warning: unknown conversion type character 'e' in format [-Wformat=]
+../../gcc/text-art/style.cc:149:25: warning: spurious leading punctuation sequence ';' in format [-Wformat-diag]
+../../gcc/text-art/style.cc:159:25: warning: spurious leading punctuation sequence ';' in format [-Wformat-diag]
+../../gcc/text-art/table.cc:979:62: warning: unquoted keyword 'char' in format [-Wformat-diag]
+../../gcc/text-art/table.cc:979:69: warning: spurious trailing punctuation sequence '])' in format [-Wformat-diag]
+./../gcc/auto-host.h:948:9: warning: "HAVE_DECL_BASENAME" redefined
+./../gcc/auto-host.h:948:9: warning: "HAVE_DECL_BASENAME" redefined
+./../gcc/auto-host.h:948:9: warning: "HAVE_DECL_BASENAME" redefined
 gengtype-lex.cc:356:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
 gengtype-lex.cc:356:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
 gengtype-lex.cc:356:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
 gengtype-lex.cc:356:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
-libtool: install: warning: remember to run `libtool --finish /usr/local/libexec/gcc/i586-alpine-linux-musl/14.0.1'
+libtool: install: warning: remember to run `libtool --finish /usr/local/libexec/gcc/i586-alpine-linux-musl/16.0.0'
diff --git a/.github/glibcxx_ubuntu64b_log_expected_warnings b/.github/glibcxx_ubuntu64b_log_expected_warnings
index 2aec0ddaff0..ba09c599d5c 100644
--- a/.github/glibcxx_ubuntu64b_log_expected_warnings
+++ b/.github/glibcxx_ubuntu64b_log_expected_warnings
@@ -63,70 +63,143 @@
 ../../../../libgcc/soft-fp/op-common.h:1563:25: warning: comparison of integer expressions of different signedness: ‘int’ and ‘USItype’ {aka ‘unsigned int’} [-Wsign-compare]
 ../../../../libgcc/soft-fp/op-common.h:1563:25: warning: comparison of integer expressions of different signedness: ‘int’ and ‘USItype’ {aka ‘unsigned int’} [-Wsign-compare]
 ../../../../libgcc/soft-fp/op-common.h:1563:25: warning: comparison of integer expressions of different signedness: ‘int’ and ‘USItype’ {aka ‘unsigned int’} [-Wsign-compare]
-../../../libcpp/expr.cc:842:35: warning: format not a string literal and no format arguments [-Wformat-security]
-../../../libcpp/expr.cc:845:38: warning: format not a string literal and no format arguments [-Wformat-security]
-../../../libcpp/expr.cc:855:33: warning: format not a string literal and no format arguments [-Wformat-security]
-../../../libcpp/expr.cc:867:42: warning: format not a string literal and no format arguments [-Wformat-security]
-../../../libcpp/expr.cc:870:39: warning: format not a string literal and no format arguments [-Wformat-security]
-../../../libcpp/expr.cc:877:35: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/expr.cc:882:35: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/expr.cc:885:38: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/expr.cc:896:33: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/expr.cc:908:42: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/expr.cc:911:39: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/expr.cc:918:40: warning: format not a string literal and no format arguments [-Wformat-security]
 ../../../libcpp/macro.cc:185:26: warning: format not a string literal and no format arguments [-Wformat-security]
 ../../../libcpp/macro.cc:214:34: warning: format not a string literal and no format arguments [-Wformat-security]
-../../../libcpp/macro.cc:3710:25: warning: format not a string literal and no format arguments [-Wformat-security]
-../../../libcpp/macro.cc:3725:25: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/analyzer/program-state.cc:1498:15: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/c-family/c-common.cc:6763:14: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/c-family/c-common.cc:6767:16: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/macro.cc:3811:25: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/macro.cc:3826:25: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/analyzer/kf-analyzer.cc:202:40: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/analyzer/kf-analyzer.cc:202:55: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/analyzer/program-state.cc:1584:15: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/attribs.cc:2330:26: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/attribs.cc:2331:44: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/attribs.cc:2332:26: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/attribs.cc:2333:44: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/attribs.cc:2352:22: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/attribs.cc:2353:40: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/attribs.cc:2354:22: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/attribs.cc:2355:40: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-common.cc:2978:13: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c-family/c-common.cc:2978:51: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-common.cc:2978:58: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-common.cc:6948:14: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/c-family/c-common.cc:6952:16: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/c-family/c-format.cc:4910:24: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c-family/c-format.cc:4910:29: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-format.cc:4910:57: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-format.cc:4911:39: warning: format ‘%d’ expects argument of type ‘int’, but argument 5 has type ‘element_format_substring*’ [-Wformat=]
+../../gcc/c-family/c-format.cc:4911:51: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-format.cc:4917:24: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c-family/c-format.cc:4917:29: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-format.cc:4917:51: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-warn.cc:3853:23: warning: format not a string literal and no format arguments [-Wformat-security]
 ../../gcc/c/c-convert.cc:84:13: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/c/c-typeck.cc:12453:16: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/c/c-typeck.cc:3878:17: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/c/c-typeck.cc:4648:16: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/c/c-typeck.cc:6913:23: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/c/c-typeck.cc:13763:16: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/c/c-typeck.cc:4445:17: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/c/c-typeck.cc:5263:16: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/c/c-typeck.cc:7557:23: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/c/c-typeck.cc:7628:11: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:7628:22: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:7628:49: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:7727:21: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:7727:23: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8577:36: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:8577:52: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8578:39: warning: format ‘%D’ expects argument of type ‘tree_node*’, but argument 4 has type ‘pp_markup::element_expected_type*’ [-Wformat=]
+../../gcc/c/c-typeck.cc:8578:65: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8583:36: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:8583:52: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8584:51: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8592:57: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8593:66: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8598:57: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8599:52: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8606:36: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:8606:71: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8607:72: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8612:36: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:8612:48: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8613:63: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8635:31: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:8635:33: warning: unknown conversion type character ‘e’ in format [-Wformat=]
 ../../gcc/collect-utils.cc:237:21: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/collect2.cc:2400:21: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/diagnostic.cc:2316:15: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/fold-const.cc:384:13: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/gcc.cc:2412:30: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2413:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2432:30: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2433:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2458:30: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2459:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2467:30: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2468:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2477:30: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2478:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2487:30: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2488:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2527:26: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2527:59: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2539:22: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2539:51: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2553:22: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2553:51: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/collect2.cc:2403:21: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/diagnostic-path.cc:1190:29: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/diagnostic-path.cc:1190:34: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/diagnostic-path.cc:1190:46: warning: format ‘%s’ expects argument of type ‘char*’, but argument 4 has type ‘element_event_desc*’ [-Wformat=]
+../../gcc/diagnostic-path.cc:1190:56: warning: format ‘%i’ expects argument of type ‘int’, but argument 5 has type ‘const char*’ [-Wformat=]
+../../gcc/diagnostic-path.cc:1196:27: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/diagnostic-path.cc:1196:32: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/diagnostic-path.cc:1196:42: warning: format ‘%i’ expects argument of type ‘int’, but argument 4 has type ‘element_event_desc*’ [-Wformat=]
+../../gcc/diagnostic-path.cc:1202:23: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/diagnostic-path.cc:1202:28: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/diagnostic.cc:1795:15: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/fold-const.cc:383:13: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/gcc.cc:2423:30: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2424:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2443:30: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2444:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2469:30: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2470:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2478:30: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2479:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2488:30: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2489:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2498:30: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2499:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2538:26: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2538:59: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2550:22: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2550:51: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2564:22: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2564:51: warning: unknown conversion type character ‘t’ in format [-Wformat=]
 ../../gcc/gimple-range-gori.cc:318:19: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/gimple-ssa-warn-access.cc:2972:20: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gimple-ssa-warn-access.cc:2972:39: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/gimple-ssa-warn-access.cc:3023:22: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gimple-ssa-warn-access.cc:3023:41: warning: unknown conversion type character ‘e’ in format [-Wformat=]
 ../../gcc/ipa-devirt.cc:967:10: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/lto-wrapper.cc:1369:10: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/lto-wrapper.cc:1971:23: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/text-art/style.cc:150:25: warning: spurious leading punctuation sequence ‘;’ in format [-Wformat-diag]
-../../gcc/text-art/style.cc:160:25: warning: spurious leading punctuation sequence ‘;’ in format [-Wformat-diag]
-../../gcc/text-art/table.cc:981:62: warning: unquoted keyword ‘char’ in format [-Wformat-diag]
-../../gcc/text-art/table.cc:981:69: warning: spurious trailing punctuation sequence ‘])’ in format [-Wformat-diag]
+../../gcc/lto-ltrans-cache.cc:164:13: warning: ‘checksum’ may be used uninitialized [-Wmaybe-uninitialized]
+../../gcc/lto-wrapper.cc:2026:23: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/opts-diagnostic.cc:173:10: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/opts-diagnostic.cc:173:10: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/opts-diagnostic.cc:174:61: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/opts-diagnostic.cc:174:61: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/opts-diagnostic.cc:254:6: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/opts-diagnostic.cc:255:53: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/opts-diagnostic.cc:357:26: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/opts-diagnostic.cc:358:69: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/pretty-print.cc:3559:19: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/pretty-print.cc:3559:28: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/pretty-print.cc:3559:38: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/pretty-print.cc:3698:19: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/pretty-print.cc:3698:28: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/pretty-print.cc:3698:38: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/rust/checks/errors/borrowck/rust-borrow-checker-diagnostics.cc:145:46: warning: narrowing conversion of ‘loan’ from ‘Rust::Polonius::Loan’ {aka ‘long unsigned int’} to ‘uint32_t’ {aka ‘unsigned int’} [-Wnarrowing]
+../../gcc/text-art/style.cc:149:25: warning: spurious leading punctuation sequence ‘;’ in format [-Wformat-diag]
+../../gcc/text-art/style.cc:159:25: warning: spurious leading punctuation sequence ‘;’ in format [-Wformat-diag]
+../../gcc/text-art/table.cc:979:62: warning: unquoted keyword ‘char’ in format [-Wformat-diag]
+../../gcc/text-art/table.cc:979:69: warning: spurious trailing punctuation sequence ‘])’ in format [-Wformat-diag]
 ../../gcc/tree-ssa-sccvn.cc:286:21: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/expr.cc:842:35: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/expr.cc:845:38: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/expr.cc:855:33: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/expr.cc:867:42: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/expr.cc:870:39: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/expr.cc:877:35: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/expr.cc:882:35: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/expr.cc:885:38: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/expr.cc:896:33: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/expr.cc:908:42: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/expr.cc:911:39: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/expr.cc:918:40: warning: format not a string literal and no format arguments [-Wformat-security]
 ../../libcpp/macro.cc:185:26: warning: format not a string literal and no format arguments [-Wformat-security]
 ../../libcpp/macro.cc:214:34: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/macro.cc:3710:25: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/macro.cc:3725:25: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/macro.cc:3811:25: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/macro.cc:3826:25: warning: format not a string literal and no format arguments [-Wformat-security]
 /home/runner/work/gccrs/gccrs/gccrs-build/x86_64-pc-linux-gnu/32/libssp/../../../../libssp/gets-chk.c:73: warning: the `gets' function is dangerous and should not be used.
 /home/runner/work/gccrs/gccrs/gccrs-build/x86_64-pc-linux-gnu/libssp/../../../libssp/gets-chk.c:73: warning: the `gets' function is dangerous and should not be used.
 gengtype-lex.cc:357:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
 gengtype-lex.cc:357:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
 gengtype-lex.cc:357:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
 gengtype-lex.cc:357:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
-install.texi:2236: warning: `.' or `,' must follow @xref, not f
-libtool: install: warning: remember to run `libtool --finish /usr/local/libexec/gcc/x86_64-pc-linux-gnu/14.0.1'
+libtool: install: warning: remember to run `libtool --finish /usr/local/libexec/gcc/x86_64-pc-linux-gnu/16.0.0'
diff --git a/.github/log_expected_warnings b/.github/log_expected_warnings
index 731fcaa556e..090e5506e29 100644
--- a/.github/log_expected_warnings
+++ b/.github/log_expected_warnings
@@ -63,92 +63,168 @@
 ../../../../libgcc/soft-fp/op-common.h:1563:25: warning: comparison of integer expressions of different signedness: ‘int’ and ‘USItype’ {aka ‘unsigned int’} [-Wsign-compare]
 ../../../../libgcc/soft-fp/op-common.h:1563:25: warning: comparison of integer expressions of different signedness: ‘int’ and ‘USItype’ {aka ‘unsigned int’} [-Wsign-compare]
 ../../../../libgcc/soft-fp/op-common.h:1563:25: warning: comparison of integer expressions of different signedness: ‘int’ and ‘USItype’ {aka ‘unsigned int’} [-Wsign-compare]
-../../../libcpp/expr.cc:842:35: warning: format not a string literal and no format arguments [-Wformat-security]
-../../../libcpp/expr.cc:845:38: warning: format not a string literal and no format arguments [-Wformat-security]
-../../../libcpp/expr.cc:855:33: warning: format not a string literal and no format arguments [-Wformat-security]
-../../../libcpp/expr.cc:867:42: warning: format not a string literal and no format arguments [-Wformat-security]
-../../../libcpp/expr.cc:870:39: warning: format not a string literal and no format arguments [-Wformat-security]
-../../../libcpp/expr.cc:877:35: warning: format not a string literal and no format arguments [-Wformat-security]
-../../../libcpp/lex.cc:2120:39: warning: ‘loc’ may be used uninitialized [-Wmaybe-uninitialized]
+../../../libcpp/expr.cc:882:35: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/expr.cc:885:38: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/expr.cc:896:33: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/expr.cc:908:42: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/expr.cc:911:39: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/expr.cc:918:40: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/lex.cc:2134:39: warning: ‘loc’ may be used uninitialized [-Wmaybe-uninitialized]
 ../../../libcpp/macro.cc:185:26: warning: format not a string literal and no format arguments [-Wformat-security]
 ../../../libcpp/macro.cc:214:34: warning: format not a string literal and no format arguments [-Wformat-security]
-../../../libcpp/macro.cc:3710:25: warning: format not a string literal and no format arguments [-Wformat-security]
-../../../libcpp/macro.cc:3725:25: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/macro.cc:3811:25: warning: format not a string literal and no format arguments [-Wformat-security]
+../../../libcpp/macro.cc:3826:25: warning: format not a string literal and no format arguments [-Wformat-security]
 ../../c++tools/server.cc:490:11: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
 ../../c++tools/server.cc:620:10: warning: ignoring return value of ‘int pipe(int*)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
-../../gcc/../libgcc/libgcov-util.c:455:9: warning: ignoring return value of ‘int chdir(const char*)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
-../../gcc/analyzer/program-state.cc:1498:15: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/analyzer/store.h:310:5: warning: ‘size_in_bytes.generic_wide_int<fixed_wide_int_storage<128> >::<anonymous>.fixed_wide_int_storage<128>::val[1]’ may be used uninitialized [-Wmaybe-uninitialized]
-../../gcc/c-family/c-common.cc:6763:14: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/c-family/c-common.cc:6767:16: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/../libgcc/libgcov-util.c:456:9: warning: ignoring return value of ‘int chdir(const char*)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
+../../gcc/analyzer/kf-analyzer.cc:202:40: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/analyzer/kf-analyzer.cc:202:55: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/analyzer/program-state.cc:1584:15: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/analyzer/store.h:312:5: warning: ‘size_in_bytes.generic_wide_int<fixed_wide_int_storage<128> >::<anonymous>.fixed_wide_int_storage<128>::val[1]’ may be used uninitialized [-Wmaybe-uninitialized]
+../../gcc/attribs.cc:2330:26: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/attribs.cc:2331:44: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/attribs.cc:2332:26: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/attribs.cc:2333:44: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/attribs.cc:2352:22: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/attribs.cc:2353:40: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/attribs.cc:2354:22: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/attribs.cc:2355:40: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-common.cc:2978:13: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c-family/c-common.cc:2978:51: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-common.cc:2978:58: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-common.cc:6948:14: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/c-family/c-common.cc:6952:16: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/c-family/c-format.cc:4910:24: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c-family/c-format.cc:4910:29: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-format.cc:4910:57: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-format.cc:4911:39: warning: format ‘%d’ expects argument of type ‘int’, but argument 5 has type ‘element_format_substring*’ [-Wformat=]
+../../gcc/c-family/c-format.cc:4911:51: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-format.cc:4917:24: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c-family/c-format.cc:4917:29: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-format.cc:4917:51: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c-family/c-warn.cc:3853:23: warning: format not a string literal and no format arguments [-Wformat-security]
 ../../gcc/c/c-convert.cc:84:13: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/c/c-typeck.cc:12453:16: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/c/c-typeck.cc:3878:17: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/c/c-typeck.cc:4648:16: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/c/c-typeck.cc:6913:23: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/c/c-typeck.cc:13763:16: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/c/c-typeck.cc:4445:17: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/c/c-typeck.cc:5263:16: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/c/c-typeck.cc:7557:23: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/c/c-typeck.cc:7628:11: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:7628:22: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:7628:49: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:7727:21: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:7727:23: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8577:36: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:8577:52: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8578:39: warning: format ‘%D’ expects argument of type ‘tree_node*’, but argument 4 has type ‘pp_markup::element_expected_type*’ [-Wformat=]
+../../gcc/c/c-typeck.cc:8578:65: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8583:36: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:8583:52: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8584:51: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8592:57: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8593:66: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8598:57: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8599:52: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8606:36: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:8606:71: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8607:72: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8612:36: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:8612:48: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8613:63: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/c/c-typeck.cc:8635:31: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/c/c-typeck.cc:8635:33: warning: unknown conversion type character ‘e’ in format [-Wformat=]
 ../../gcc/collect-utils.cc:237:21: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/collect2.cc:2400:21: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/diagnostic.cc:2316:15: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/expmed.cc:1845:45: warning: ‘*(unsigned int*)((char*)&imode + offsetof(scalar_int_mode, scalar_int_mode::m_mode))’ may be used uninitialized in this function [-Wmaybe-uninitialized]
-../../gcc/fold-const.cc:384:13: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/gcc.cc:10668:12: warning: ignoring return value of ‘ssize_t read(int, void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
-../../gcc/gcc.cc:2412:30: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2413:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2432:30: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2433:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2458:30: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2459:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2467:30: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2468:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2477:30: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2478:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2487:30: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2488:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2527:26: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2527:59: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2539:22: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2539:51: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:2553:22: warning: too many arguments for format [-Wformat-extra-args]
-../../gcc/gcc.cc:2553:51: warning: unknown conversion type character ‘t’ in format [-Wformat=]
-../../gcc/gcc.cc:7897:9: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
-../../gcc/gcc.cc:7900:13: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
-../../gcc/gcc.cc:7901:13: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
-../../gcc/gcc.cc:7903:9: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
+../../gcc/collect2.cc:2403:21: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/diagnostic-path.cc:1190:29: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/diagnostic-path.cc:1190:34: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/diagnostic-path.cc:1190:46: warning: format ‘%s’ expects argument of type ‘char*’, but argument 4 has type ‘element_event_desc*’ [-Wformat=]
+../../gcc/diagnostic-path.cc:1190:56: warning: format ‘%i’ expects argument of type ‘int’, but argument 5 has type ‘const char*’ [-Wformat=]
+../../gcc/diagnostic-path.cc:1196:27: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/diagnostic-path.cc:1196:32: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/diagnostic-path.cc:1196:42: warning: format ‘%i’ expects argument of type ‘int’, but argument 4 has type ‘element_event_desc*’ [-Wformat=]
+../../gcc/diagnostic-path.cc:1202:23: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/diagnostic-path.cc:1202:28: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/diagnostic.cc:1795:15: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/expmed.cc:1864:45: warning: ‘*(unsigned int*)((char*)&imode + offsetof(scalar_int_mode, scalar_int_mode::m_mode))’ may be used uninitialized in this function [-Wmaybe-uninitialized]
+../../gcc/fold-const.cc:383:13: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/gcc.cc:10706:12: warning: ignoring return value of ‘ssize_t read(int, void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
+../../gcc/gcc.cc:2423:30: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2424:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2443:30: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2444:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2469:30: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2470:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2478:30: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2479:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2488:30: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2489:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2498:30: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2499:32: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2538:26: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2538:59: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2550:22: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2550:51: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:2564:22: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gcc.cc:2564:51: warning: unknown conversion type character ‘t’ in format [-Wformat=]
+../../gcc/gcc.cc:7938:9: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
+../../gcc/gcc.cc:7941:13: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
+../../gcc/gcc.cc:7942:13: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
+../../gcc/gcc.cc:7944:9: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
+../../gcc/gcc.cc:9325:13: warning: ignoring return value of ‘int truncate(const char*, __off_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
 ../../gcc/genhooks.cc:112:17: warning: ignoring return value of ‘int fscanf(FILE*, const char*, ...)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
 ../../gcc/genhooks.cc:121:14: warning: ignoring return value of ‘int fscanf(FILE*, const char*, ...)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
 ../../gcc/genhooks.cc:174:14: warning: ignoring return value of ‘int fscanf(FILE*, const char*, ...)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
 ../../gcc/genhooks.cc:180:14: warning: ignoring return value of ‘int fscanf(FILE*, const char*, ...)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
+../../gcc/gimple-fold.cc:7677:20: warning: ‘xor_cmp_op’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 ../../gcc/gimple-range-gori.cc:318:19: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/gimple-ssa-warn-access.cc:2972:20: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gimple-ssa-warn-access.cc:2972:39: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/gimple-ssa-warn-access.cc:3023:22: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/gimple-ssa-warn-access.cc:3023:41: warning: unknown conversion type character ‘e’ in format [-Wformat=]
 ../../gcc/ipa-devirt.cc:967:10: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/lto-wrapper.cc:1151:8: warning: ignoring return value of ‘ssize_t read(int, void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
-../../gcc/lto-wrapper.cc:1369:10: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/lto-wrapper.cc:1971:23: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/lto/lto-common.cc:2084:10: warning: ignoring return value of ‘int fscanf(FILE*, const char*, ...)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
-../../gcc/lto/lto-common.cc:2086:9: warning: ignoring return value of ‘size_t fread(void*, size_t, size_t, FILE*)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
-../../gcc/lto/lto-common.cc:2106:10: warning: ignoring return value of ‘int fscanf(FILE*, const char*, ...)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
-../../gcc/text-art/style.cc:150:25: warning: spurious leading punctuation sequence ‘;’ in format [-Wformat-diag]
-../../gcc/text-art/style.cc:160:25: warning: spurious leading punctuation sequence ‘;’ in format [-Wformat-diag]
-../../gcc/text-art/table.cc:981:62: warning: unquoted keyword ‘char’ in format [-Wformat-diag]
-../../gcc/text-art/table.cc:981:69: warning: spurious trailing punctuation sequence ‘])’ in format [-Wformat-diag]
+../../gcc/lto-wrapper.cc:1171:8: warning: ignoring return value of ‘ssize_t read(int, void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
+../../gcc/lto-wrapper.cc:2026:23: warning: format not a string literal and no format arguments [-Wformat-security]
+../../gcc/lto/lto-common.cc:2095:10: warning: ignoring return value of ‘int fscanf(FILE*, const char*, ...)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
+../../gcc/lto/lto-common.cc:2097:9: warning: ignoring return value of ‘size_t fread(void*, size_t, size_t, FILE*)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
+../../gcc/lto/lto-common.cc:2117:10: warning: ignoring return value of ‘int fscanf(FILE*, const char*, ...)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
+../../gcc/opts-diagnostic.cc:173:10: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/opts-diagnostic.cc:173:10: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/opts-diagnostic.cc:174:61: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/opts-diagnostic.cc:174:61: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/opts-diagnostic.cc:254:6: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/opts-diagnostic.cc:255:53: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/opts-diagnostic.cc:357:26: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/opts-diagnostic.cc:358:69: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/pretty-print.cc:3559:19: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/pretty-print.cc:3559:28: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/pretty-print.cc:3559:38: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/pretty-print.cc:3698:19: warning: too many arguments for format [-Wformat-extra-args]
+../../gcc/pretty-print.cc:3698:28: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/pretty-print.cc:3698:38: warning: unknown conversion type character ‘e’ in format [-Wformat=]
+../../gcc/rust/checks/errors/borrowck/rust-borrow-checker-diagnostics.cc:145:46: warning: narrowing conversion of ‘loan’ from ‘Rust::Polonius::Loan’ {aka ‘long unsigned int’} to ‘uint32_t’ {aka ‘unsigned int’} [-Wnarrowing]
+../../gcc/text-art/style.cc:149:25: warning: spurious leading punctuation sequence ‘;’ in format [-Wformat-diag]
+../../gcc/text-art/style.cc:159:25: warning: spurious leading punctuation sequence ‘;’ in format [-Wformat-diag]
+../../gcc/text-art/table.cc:979:62: warning: unquoted keyword ‘char’ in format [-Wformat-diag]
+../../gcc/text-art/table.cc:979:69: warning: spurious trailing punctuation sequence ‘])’ in format [-Wformat-diag]
 ../../gcc/tree-ssa-sccvn.cc:286:21: warning: format not a string literal and no format arguments [-Wformat-security]
-../../gcc/tree-vect-stmts.cc:10397:47: warning: ‘stride_step’ may be used uninitialized in this function [-Wmaybe-uninitialized]
-../../gcc/vec.h:347:10: warning: ‘void free(void*)’ called on unallocated object ‘dest_bbs’ [-Wfree-nonheap-object]
-../../libcpp/expr.cc:842:35: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/expr.cc:845:38: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/expr.cc:855:33: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/expr.cc:867:42: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/expr.cc:870:39: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/expr.cc:877:35: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/lex.cc:2120:39: warning: ‘loc’ may be used uninitialized [-Wmaybe-uninitialized]
+../../gcc/tree-vect-stmts.cc:10897:47: warning: ‘stride_step’ may be used uninitialized in this function [-Wmaybe-uninitialized]
+../../gcc/tree.h:3838:41: warning: ‘append_args_last’ may be used uninitialized in this function [-Wmaybe-uninitialized]
+../../gcc/vec.h:347:10: warning: ‘void free(void*)’ called on unallocated object ‘tombstone_uids’ [-Wfree-nonheap-object]
+../../gcc/vec.h:347:10: warning: ‘void free(void*)’ called on unallocated object ‘unique_labels’ [-Wfree-nonheap-object]
+../../libcpp/expr.cc:882:35: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/expr.cc:885:38: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/expr.cc:896:33: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/expr.cc:908:42: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/expr.cc:911:39: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/expr.cc:918:40: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/lex.cc:2134:39: warning: ‘loc’ may be used uninitialized [-Wmaybe-uninitialized]
 ../../libcpp/macro.cc:185:26: warning: format not a string literal and no format arguments [-Wformat-security]
 ../../libcpp/macro.cc:214:34: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/macro.cc:3710:25: warning: format not a string literal and no format arguments [-Wformat-security]
-../../libcpp/macro.cc:3725:25: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/macro.cc:3811:25: warning: format not a string literal and no format arguments [-Wformat-security]
+../../libcpp/macro.cc:3826:25: warning: format not a string literal and no format arguments [-Wformat-security]
 /home/runner/work/gccrs/gccrs/gccrs-build/x86_64-pc-linux-gnu/32/libssp/../../../../libssp/gets-chk.c:73: warning: the `gets' function is dangerous and should not be used.
 /home/runner/work/gccrs/gccrs/gccrs-build/x86_64-pc-linux-gnu/libssp/../../../libssp/gets-chk.c:73: warning: the `gets' function is dangerous and should not be used.
 gengtype-lex.cc:357:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
 gengtype-lex.cc:357:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
 gengtype-lex.cc:357:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
 gengtype-lex.cc:357:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
-install.texi:2236: warning: `.' or `,' must follow @xref, not f
-libtool: install: warning: remember to run `libtool --finish /usr/local/libexec/gcc/x86_64-pc-linux-gnu/14.0.1'
+libtool: install: warning: remember to run `libtool --finish /usr/local/libexec/gcc/x86_64-pc-linux-gnu/16.0.0'
-- 
2.48.1

